client/src/components/Tags/Tags.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Tags.scss';
+
+/**
+ * Renders a list of tags belonging to a Monument
+ */
+export default class Tags extends React.Component {
+
+    render() {
+        if (!this.props.tags) return (<div/>);
+        return (
+            <div className="tags">
+                {this.props.tags.sort((a, b) => a.name.length - b.name.length).map(tag => {
+                    return (
+                        <div key={tag.name} className="tag text-truncate">{tag.name}</div>
+                    )
+                })}
+            </div>
+        );
+    }
+}
```

Comment:
Would it ever be useful to split each Tag into its own Component?

Reply:
I was going to do that originally but my guess is that it probably adds some overhead to the rendering that isn't really necessary since the tag itself is very simplistic and can just be handled by Tags. Not sure how true that really is

Reply:
Not a big deal, we'll split it into its own Component if we need to.